tests/mocha/generator_test.js

```diff
@@ -7,11 +7,11 @@
 goog.declareModuleId('Blockly.test.generator');
 
 import * as Blockly from '../../build/src/core/blockly.js';
-const {dartGenerator} = goog.require('Blockly.Dart');
-const {javascriptGenerator} = goog.require('Blockly.JavaScript');
-const {luaGenerator} = goog.require('Blockly.Lua');
-const {phpGenerator} = goog.require('Blockly.PHP');
-const {pythonGenerator} = goog.require('Blockly.Python');
+const {DartGenerator} = goog.require('Blockly.Dart.all');
```

Comment:
Why is this one different than the others? iiuc the `all` shouldn't be here since that one doesn't have the constructor.

Reply:
Good catch.  You are quite correct.  Fixed.